test/openlr.cc

```diff
@@ -52,26 +36,28 @@ struct testfixture {
   double expectedNoff;
 };
 
-const testfixture testfixtures[] = {{"CwOa9yUQACODBQEqAL4jEw==", 5.069987, 52.119130, 39.375,
-                                     5.072967, 52.121030, 219.375, 293, 0, 0},
-                                    {"CwOa9yUQACODqgEqAL4jEw==", 5.069987, 52.119130, 39.375,
-                                     5.072967, 52.121030, 219.375, 9962, 0, 0},
-                                    {"CwOiYCUMoBNWAv9P/+MSBg==", 5.110692, 52.100590, 253.125,
-                                     5.108922, 52.100300, 73.125, 117.2, 0, 0},
-                                    {"CxWj2OogyxJBDhDSAvwSUL4=", 30.431259, -30.757352, 16.875,
-                                     30.474319, -30.749712, 185.625, 820.4, 190, 0},
-                                    {"CxWj2OogyxJBDhDSAvwSMA0=", 30.431259, -30.757352, 16.875,
-                                     30.474319, -30.749712, 185.625, 820.4, 0, 13},
-                                    {"CxWj2OogyxJBDhDSAvwScL4N", 30.431259, -30.757352, 16.875,
-                                     30.474319, -30.749712, 185.625, 820.4, 190, 13},
-                                    {"C6i8rRtM3BjgAAAAAAUYAA==", -122.713562, 38.390942, 5.625,
-                                     -122.713562, 38.390991, 5.625, 0, 0, 0}};
+const testfixture testfixtures[] = {
+    {"CwOa9yUQACODBQEqAL4jEw==", 5.069987, 52.119130, 39.375, 5.072967, 52.121030, 219.375, 293, 0,
+     0},
+    {"CwOa9yUQACODqgEqAL4jEw==", 5.069987, 52.119130, 39.375, 5.072967, 52.121030, 219.375, 9962, 0,
+     0},
+    {"CwOiYCUMoBNWAv9P/+MSBg==", 5.110692, 52.100590, 253.125, 5.108922, 52.100300, 73.125, 117.2, 0,
+     0},
+    {"CxWj2OogyxJBDhDSAvwSUL4=", 30.431259, -30.757352, 16.875, 30.474319, -30.749712, 185.625, 820.4,
+     190, 0},
+    {"CxWj2OogyxJBDhDSAvwSMA0=", 30.431259, -30.757352, 16.875, 30.474319, -30.749712, 185.625, 820.4,
+     0, 13},
+    {"CxWj2OogyxJBDhDSAvwScL4N", 30.431259, -30.757352, 16.875, 30.474319, -30.749712, 185.625, 820.4,
+     190, 13},
+    {"C6i8rRtM3BjgAAAAAAUYAA==", -122.713562, 38.390942, 5.625, -122.713562, 38.390991, 5.625, 0, 0,
+     0},
```

Comment:
this is just the comma on the end changing the formatter right? my eyes can see any other change in the data but its very hard to tell with the numbers shifted

Reply:
Yep, the addition of the comma here changes the formatting.